.github/scripts/check-maven-for-llvm-firtool-version.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# Strip firtool- prefix from argument
+VERSION=${1#firtool-}
+
+POM="https://repo1.maven.org/maven2/org/chipsalliance/llvm-firtool/$VERSION/llvm-firtool-$VERSION.pom"
+
+# Return 1 if not found
+curl -o/dev/null -sfIL $POM || false
```

Comment:
Nit: double quote this:

```suggestion
curl -o/dev/null -sfIL "$POM" || false
```